Kickstarter-iOS/Views/Cells/PledgeAmountCell.swift

```diff
@@ -80,13 +86,45 @@ final class PledgeAmountCell: UITableViewCell, ValueCell {
 
     self.amountInputView.label.rac.text = self.viewModel.outputs.currency
     self.amountInputView.textField.rac.text = self.viewModel.outputs.amount
+
+    self.viewModel.outputs.stepperInitialValue
```

Comment:
I will update `ReactiveExtensions` project to have the ability to use `rac` assignments. Maybe in a separate PR?